configmap/parse.go

```diff
@@ -144,6 +146,45 @@ func AsQuantity(key string, target **resource.Quantity) ParseFunc {
 	}
 }
 
+func AsNamespacedName(key string, target *types.NamespacedName) ParseFunc {
+	return func(data map[string]string) error {
+		var (
+			name      string
+			namespace string
+			raw       string
+			ok        bool
+		)
+
+		if raw, ok = data[key]; !ok {
```

Comment:
Drop declaration above and use:
```suggestion
		raw, ok := data[key]
		if !ok {
```